src/test/java/pages/HomePage.java

```diff
@@ -0,0 +1,109 @@
+package pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+public class HomePage extends BasePage {
+
+    public HomePage(WebDriver givenDriver) {
+
+        super(givenDriver);
+    }
+    @FindBy(xpath ="//img[@class='avatar']")
+    private WebElement avatarIcon;
+     @FindBy( xpath ="//a[@class='songs']")
+     private WebElement allSongsList;
+
+     @FindBy (css ="[data-testid='play-btn]")
+     private WebElement playBtn;
+
+     @FindBy (xpath = "[//button[@data-test='view-all-songs-btn']")
+     private WebElement viewAllBtn;
+
+     @FindBy (css ="[.playlist:nth-child(3)]")
+     private WebElement firstPlaylist;
+
+     @FindBy (css="[name='name']")
+     private WebElement editPlaylistNameField;
+
+     @FindBy (xpath ="[//section[@id='songResultsWrapper']")
+     private WebElement songResultsWrapper;
+
+     @FindBy(css = "[type='search']")
+     private WebElement searchField;
+
+     @FindBy(xpath = "//button[@class='del btn-delete-playlist']")
+     private WebElement deletePlaylistBtn;
+
+     @FindBy(xpath = "//button[@class='ok']")
+     private WebElement confirmDeleteBtn;
+
+    private WebElement getAvatarIcon() {
+
+       return avatarIcon;
+    }
+
+    public void clickViewAllBtn() {
+            //waitForOverlay();
+            viewAllBtn.click();
+
+        }
+    public void choosePlaylist() {
+
+        firstPlaylist.click();
+    }
+
+    public void chooseAllSongsList() {
+      // waitForOverlay();
+        allSongsList.click();
+    }
+
+
+        public void searchForSong(String song){
+       // waitForOverlay();
+            searchField.click();
+            searchField.clear();
+            searchField.sendKeys(song);
+
+        }
+    public void editPlaylistName(String playListName) {
+        editPlaylistNameField.sendKeys(Keys.chord(Keys.CONTROL,"A",Keys.BACK_SPACE));
+        editPlaylistNameField.sendKeys(playListName);
+        editPlaylistNameField.sendKeys(Keys.ENTER);
+    }
+
+
+
+    public void doubleClickChoosePlaylist() {
+
+        actions.doubleClick(firstPlaylist).click().perform();
+    }
+    public void rightClickPlaylist () {
+
+        actions.contextClick(firstPlaylist).click().perform();
+    }
+    public void clickDeleteBtn() {
+        deletePlaylistBtn.click();
+
+        confirmDeleteBtn.click();
+    }
+    public void getDeletionText(){
+
+        confirmationMessage.getText();
```

Comment:
You got this problem, please correct it. 

![image](https://github.com/Test-Pro-Bootcamp/A47-QA-Automation-Framework/assets/104761482/2ed9f17c-83e4-40af-a6be-4648e7eae7d7)

HINT

Add 

```
   @FindBy(xpath = "[//div[@class='alertify-logs top right']")
    private WebElement confirmationMessage;
```
![image](https://github.com/Test-Pro-Bootcamp/A47-QA-Automation-Framework/assets/104761482/42f152ae-8bbb-482a-9729-b1455fde97be)